pkg/virt-launcher/virtwrap/nichotplug.go

```diff
@@ -95,6 +97,40 @@ func (vim *virtIOInterfaceManager) hotplugVirtioInterface(vmi *v1.VirtualMachine
 	return nil
 }
 
+func (vim *virtIOInterfaceManager) updateDomainLinkStates(currentDomain, desiredDomain *api.Domain) error {
```

Comment:
I find the logic of changing the link-state as not being related to `nichotplug`.
However, I do not have an immediate alternative.